ee_plugin/Makefile

```diff
@@ -23,37 +23,47 @@ PY_FILES = \
 
 UI_FILES = ee_dock_widget_base.ui
 
-EXTRAS = metadata.txt icons/earth_engine.svg
+EXTRAS = metadata.txt home_plugin.html
 
-EXTRA_DIRS =
+EXTRA_DIRS = icons
 
 COMPILED_RESOURCE_FILES = resources.py
 
 PEP8EXCLUDE=pydev,resources.py,conf.py,third_party,ui
 
+# QGISDIR points to the location where your plugin should be installed.
+# This varies by platform, relative to your HOME directory:
+#	* Linux:
+#	  .local/share/QGIS/QGIS3/profiles/default
+#	* Mac OS X:
+#	  Library/Application Support/QGIS/QGIS3/profiles/default
+#	* Windows:
+#	  AppData\Roaming\QGIS\QGIS3\profiles\default'
+
+QGISDIR=.local/share/QGIS/QGIS3/profiles/default
```

Comment:
This .local will give issues on Windows?

Reply:
mmm this requires some OS conditional for set this variable...